app/routes/cases/index.js

```diff
@@ -33,9 +33,9 @@ export default class CasesIndexRoute extends Route {
       },
     };
     if (params.showArchivedOnly) {
-      options['filter[case][is-archived]'] = true;
+      options['filter[:has:closed]'] = true;
     } else {
-      options['filter[case][is-archived]'] = false;
+      options['filter[:has-no:closed]'] = true;
     }
```

Comment:
On normal queries we also shouldn't use `:has:` and `:has-no:` on properties since that is not really supported.
We should either check if there is a date or nothing.

Reply:
seems to work. Might not work 100% of the time?
I remember having a conversation that this was not really supported but was a long time ago.
Leaving it as-is for now.